master/security/service-accounts.md

```diff
@@ -0,0 +1,91 @@
+
+### Value
+
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the network security can have RBAC permissions to:
+
+- Control which service accounts the developer team can use within a namespace
+- Write high-priority network policies for those service accounts (that the developer team cannot override) 
+
+The network security team can maintain full control of security, while selectively allowing developer operations where it makes sense.  
+
+Using **Istio-enabled apps** with Calico network policy, the cryptographic identity associated with the service account is checked (along with the network identity) to achieve two-factor authentication.
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+**NetworkPolicy** or **GlobalNetworkPolicy** with a service account rule and match criteria.
+
+
+### Concepts
+
+#### Use smallest set of permissions required
+
+Service accounts are controlled by RBAC so you can grant permissions to trusted entities (code and/or people). To perform any operation in a workload, clients are required to authenticate with the Kubernetes API server. If you do not explicitly assign a service account to a pod, it uses the default ServiceAccount in the namespace. You should not grant broad permissions to the default service account for a namespace. If an application needs access to the Kubernetes API, create separate service accounts with the smallest set of permissions required. 
+
+#### Service account labels
+
+Like all other Kubernetes objects, service accounts have labels. You can use labels to create ‘groups’ of service accounts. Calico network policy lets you select workloads by their service account using:
+
+- An exact match on service account name
+- A service account label selector expression
+
+### Before you begin...
+
+Configure unique Kubernetes service accounts for your applications
+
+### How to
+
+- [Limit ingress traffic for workloads by service account name](#limit-ingress-traffic-for-workloads-by-service-account-name)
+- [Limit ingress traffic for workloads by service account label](#limit-ingress-traffic-for-workloads-by-service-account-label)
+
+#### Limit ingress traffic for workloads by service account name
+
+In the following example, ingress traffic is allow from any workload whose service account matches the names **api-service** or **use-auth-service**.
+
+```
+apiVersion: projectcalico.org/v3
+kind: NetworkPolicy
+metadata: 
+  name: demo-calico
+  namespace: prod-engineering
+spec: 
+  ingress: 
+    - action: Allow
+      source: 
+        serviceAccounts: 
+          names: 
+            - api-service
+            - use-auth-service
+```
+
+#### Limit ingress traffic for workloads by service account label
+
+In the following example, ingress traffic is allowed from any workload whose service account matches the label, **app == web-frontend**.
```

Comment:
I'd say "matches the label _selector_" here (subtle difference between a label and a selector - the thing written here is a selector rather than a label).

Reply:
^ Comment still needs to be addressed

Reply:
Fixed.